nawab.py

```diff
@@ -46,25 +47,58 @@ def wrapper(func, args, res):
 
 
 def main():
+    parser = argparse.ArgumentParser(add_help=False)
+    parser.add_argument("-r", "--retweet", help="Retweet all tweets automatically, doesn't spawn a telegram bot",
+                        action='store_true', required=False)
+    parser.add_argument("-t", "--tg_bot", help="Doesn't retweet all automatically, can manually retweet through telegram bot",
+                        action='store_true', required=False)
+    parser.add_argument("-b", "--blacklist",type=list, required=False, help="Blacklist the given username")
+    parser.add_argument("-p", "--path", type=str, required=False,
+                        help="Path where the log files be stored. Note to create directory in that path beforehand.")
+    parser.add_argument('-d', '--default', action="store_const", const=30)
+    parser.add_argument('-V', '--verbose',action="store_const", const=20)
+    parser.add_argument('-s', '--silent', action="store_const", const=50)
+    parser.add_argument('-h', '--help', action='help', default=argparse.SUPPRESS,
+                        help='Show this help message and exit.')
+    parser.add_argument('-v', '--version', action='version',
+                        version='%(prog)s 1.0', help="Show program's version number and exit.")
+
+    args = vars(parser.parse_args())
+    ## set manual loglevels
+    if args['default']:
+        level = args['default']
+    elif args['verbose']:
+        level = args['verbose']
+    elif args['silent']:
+        level = args['silent']
+    else:
+        level = 10
+    names = args['blacklist']
+
     res = []
     data = pd.read_csv('data.csv')
     default_dir = '/var/log/nawab/'
+    ## blacklisting the names to current data.csv files
+    if names:
+        blist = pd.DataFrame({'Blacklist': names})
```

Comment:
isn't it better to say, 
if args['blacklist']:
     blist = pd.DataFrame({'Blacklist': args['blacklist']})